modules/lib/src/main/resources/assets/js/app/security/UserAccessListItemView.ts

```diff
@@ -44,7 +44,9 @@ export class UserAccessListItemView
 
             this.resizeListener = this.setExtraCount.bind(this);
             ResponsiveManager.onAvailableSizeChanged(this, this.resizeListener);
```

Comment:
Seems we also don't need this and overridden `remove()` method :)
Responsive manager is also adding some classes describing size (in addition to reacting to size change), but they are not used in styling, thus we can remove it